recipe/meta.yaml

```diff
@@ -1,43 +1,81 @@
 {% set name = "tiledbsoma" %}
-{% set version = "0.5.0a2" %}
+{% set version = "1.0rc0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e0fb16e55e01315a46f102739dcd3d1b15f24d636e00a2d737c0dde674b86370
-
-build:
-  number: 0
-  script: python -m pip install . -vv
-  skip: true  # [win]
+  git_url: https://github.com/single-cell-data/TileDB-SOMA.git
+  git_rev: 8a57a05edecc2b2e17675dc59791c2dedc764034
 
 requirements:
```

Comment:
You can simplify recipe maintenance and the build process by removing these top-level requirements and combining `build.sh` and `build-libtiledbsome.sh`. My PR https://github.com/TileDB-Inc/tiledb-vcf-feedstock/pull/75 implemented this for the libtiledbvcf recipe